bitcoin/electrs/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:alpine3.13
+
+RUN apk add clang-dev build-base linux-headers git
+RUN rustup component add rustfmt
+WORKDIR /workdir
+RUN git clone https://github.com/Blockstream/electrs.git
```

Comment:
I'm wondering how will it behave when the referenced repo gets updated, probably we will have to rebuild all the layers even if we want just a specific commit. Building of electrs takes a really long time. Ideally I would use a clone of a specific commit hash but I don't see it's possible.
Maybe we could do 
```Dockerfile
RUN git clone --no-checkout https://github.com/Blockstream/electrs.git
WORKDIR electrs
RUN git checkout a33e97e1a1fc63fa9c20a116bb92579bbf43b254
```
?